Dockerfile

```diff
@@ -5,16 +5,16 @@
 # See https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation
 FROM mcr.microsoft.com/dotnet/sdk:7.0-alpine AS build-env
 
-ARG MS_NUGET_URL=https://nuget.pkg.github.com/microsoft/index.json
 ARG MSGRAPH_NUGET_URL=https://nuget.pkg.github.com/microsoftgraph/index.json
 
 WORKDIR /app
 
 COPY ./src ./msgraph-cli/src
 WORKDIR /app/msgraph-cli
 
+# RUN apk add gcc --no-cache # Uncomment this if PublishAot is enabled
+
 RUN --mount=type=secret,id=user,required=true --mount=type=secret,id=token,required=true \
-    dotnet nuget add source ${MS_NUGET_URL} -n ms-gh -u $(cat /run/secrets/user) -p $(cat /run/secrets/token) --store-password-in-clear-text &&\
     dotnet nuget add source ${MSGRAPH_NUGET_URL} -n msgraph-gh -u $(cat /run/secrets/user) -p $(cat /run/secrets/token) --store-password-in-clear-text
```

Comment:
Should we also remove this line as we don't have packages in private feeds anymore?

Reply:
Graph CLI core is still on a private feed